qtgui: Fixes #5623 - QEngValidator in range.py.cmakein incorrectly overwrites default background color

jb41997 opened the pull request:
## Description
QEngValidator in range.py.cmakein incorrectly overwrites default background color to white on successful validation. White may not be the correct color given the default qss of the user. Fix changes it to default coloring instead of white.

## Related Issue
Fixes #5623

## Which blocks/areas does this affect?
QT GUI Range

## Testing Done
Tested on osmocom_fft on kali linux OS.

## Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/gnuradio/gnuradio/blob/main/CONTRIBUTING.md).
- [x] I have squashed my commits to have one significant change per commit. 
- [x] I [have signed my commits before making this PR](https://github.com/gnuradio/gnuradio/blob/main/CONTRIBUTING.md#dco-signed)
- [x] My code follows the code style of this project. See [GREP1.md](https://github.com/gnuradio/greps/blob/main/grep-0001-coding-guidelines.md).
- [ ] I have updated [the documentation](https://wiki.gnuradio.org/index.php/Main_Page#Documentation) where necessary.
- [ ] I have added tests to cover my changes, and all previous tests pass.

willcode reviewed (approved):
Looks right. Additionally, when the `background-color` is set to `yellow`, `color` should be set to `black` in case the theme has white text.

jb41997 commented:
Whew, apologies for the mess. I had some issues with my signature failing the DCO check. I fixed that I think. Then fixed the "color: black" on the yellow backgrounds.  I'm still seeing a DCO failure for some reason even though I've fixed my signature. Thoughts?

willcode commented:
Check the indent on line 61.

Could you squash these commits into one? That may also help with the signoff that doesn't seem to be working right. Not exactly sure why it's not being detected.

Also, the git history should stand alone without github, so no need to put issue numbers in the commit message, just reference it in the PR.

jb41997 commented:
@willcode I think I squashed it...finally. Also fixed some formatting. Learning vim so it took a few tries.

willcode commented:
Getting there. You picked up an unrelated commit in the process of squashing. The formatting will probably not pass (see tools/check_pep8.sh and tools/autoformat_python.sh).

willcode commented:
I rebased/squashed and pushed the fixed-up commit to your branch. Should be all set now.

jb41997 commented:
@willcode sounds good. Man, i got stuck in rebase hell. Learned a lot though!  Hopefully the next one goes a little smoother...
